lib/ruby-push-notifications/mpns/mpns_notification.rb

```diff
@@ -0,0 +1,96 @@
+require 'builder'
+
+module RubyPushNotifications
+  module MPNS
+    # Encapsulates a MPNS Notification.
+    # Actually support for raw, toast, tiles notifications
+    # (http://msdn.microsoft.com/en-us/library/windowsphone/develop/hh202945)
+    #
+    class MPNSNotification
+
+      # @return [Array]. Array with the results from sending this notification.
+      attr_accessor :results
```

Comment:
Results is not an array, but it should be to show same behaviour as other notification types (iOS, Android). I'd suggest having the pusher do the parsing from the response hashes (`{ device_url: url.to_s, headers: response.to_hash, code: response.code.to_i }`) into `MPNSResult` objects. And then assign here the array of those results.